src/main/java/com/softserve/paymentservice/exceptionHandler/PaymentExceptionHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.softserve.paymentservice.exceptionHandler;
+
+import com.softserve.paymentservice.model.ErrorResponse;
+import com.stripe.exception.StripeException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.time.LocalDateTime;
+
+@RestControllerAdvice// todo @responseStatus above mycustomEcxception --> we don't need this
+public class PaymentExceptionHandler extends ResponseEntityExceptionHandler {
+
+
+    @ExceptionHandler(StripeException.class)
+    public ResponseEntity<ErrorResponse> stripeException(Exception e) {
+        ErrorResponse errorResponse = ErrorResponse.builder()
+                .time(LocalDateTime.now())
+                .status(500)
+                .error(e.getMessage())
+                .build();
+        return new ResponseEntity<>(errorResponse, HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+
+    //    @ExceptionHandler(value = {CardException.class})
```

Comment:
Vadim said to not pushing commits with commented blocks of code

Reply:
If you need to save this code for further tasks:
* put the code in the ticket
* move it to the separate branch